driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverRequest.java

```diff
@@ -54,11 +54,11 @@ public URL getSeleniumServerUrl() {
     }
 
     public DesktopWebDriverRequest setSeleniumServerUrl(String url) throws MalformedURLException {
-        this.seleniumServerURL = new URL(url);
-        return this;
+        return this.setSeleniumServerUrl(new URL(url));
     }
 
     public DesktopWebDriverRequest setSeleniumServerUrl(URL url) {
+        this.setWebDriverMode(WebDriverMode.remote);
```

Comment:
From my point this does not make sense here:

`DesktopWebDriverFactory (295/296)`: In case of `desktopWebDriverRequest.getWebDriverMode() == WebDriverMode.remote` the method `DesktopWebDriverRequest.getSeleniumServerUrl` is called.
--> Within the method `DesktopWebDriverRequest.getSeleniumServerUrl` the method `DesktopWebDriverRequest.setSeleniumServerUrl` is called where mode is switched again to `remote`.

Reply:
I guess it doesn't make sense to have a `tt.webdriver.mode`

Reply:
Why not? The Testerra tests are running with `tt.webdriver.mode=local` without any Selenium server

Reply:
Yes, but having a configured Selenium server URL would be enough.
`tt.selenium.server.url=` -> local
`tt.selenium.server.url=http://` -> remote

Reply:
From user's view there is no change, but from API's view it now easier.